plugins/acl/project-role-based/src/main/java/org/apache/cloudstack/acl/ProjectRoleBasedApiAccessChecker.java

```diff
@@ -58,9 +58,13 @@ private void denyApiAccess(final String commandName) throws PermissionDeniedExce
         throw new PermissionDeniedException("The API " + commandName + " is denied for the user's/account's project role.");
     }
 
+    @Override
+    public boolean isEnabled() {
+        return roleService.isEnabled();
+    }
 
     public boolean isDisabled() {
```

Comment:
Not an issue but an improvement request - can we leave only one of these methods?

Reply:
both are used, so no ;)